prod-buildspec.yml

```diff
@@ -0,0 +1,126 @@
+version: 0.2
```

Comment:
This is the wrong buildspec. We are not using batch builds, that was a short lived experiment.